### PR TITLE
Render OpenID grid using client

### PR DESCRIPTION
**Pull request by guerler**
This PR renders the mako-based OpenID grid using the front-end client. It would be nice if someone with more OpenID experience can take a look and verify that the process still works smoothly. ping @VJalili.

### CODE REVIEW

**Comment by guerler** [comment]
@galaxybot test this.

**Comment by guerler** [comment]
@galaxybot test this

File: lib/galaxy/webapps/galaxy/controllers/user.py [hunk 1]
```diff
@@ -386,63 +381,53 @@ def openid_associate(self, trans, cntrller='user', **kwd):
...
-                for deleted_url in deleted_urls:
-                    trans.log_event("User disassociated OpenID: %s" % deleted_url)
-                message = '%s OpenIDs were disassociated from your Galaxy account.' % len(ids)
-                status = 'done'
-        return trans.response.send_redirect(url_for(controller='user',
-                                                    action='openid_manage',
-                                                    use_panels=use_panels,
-                                                    message=message,
-                                                    status=status))
+    @web.require_admin
```
Comment (VJalili) [inline lib/galaxy/webapps/galaxy/controllers/user.py:384]: Any user should be able to add an OpenID token to his/her account. I'm not sure if we want to make it available to admins only. Basically with this change only admins are allowed to add OpenID token to their account, and non-admin users are prevented. Is there any specific reason for it ?

**Review by VJalili — CHANGES_REQUESTED** [review]

**Comment by VJalili** [comment]
I was able to test this with AOL, and it works fine, except for the admin requirement. 

Incidentally, Yahoo has suspended OpenID support, so I removed its backend (see PR #5208)

File: lib/galaxy/webapps/galaxy/controllers/user.py (hunk 1 above)
Comment (guerler) [reply lib/galaxy/webapps/galaxy/controllers/user.py:384]: Thanks VJalili. I'll fix that.

**Comment by guerler** [comment]
Merged this into #5181.

**Pull request closed without merging**